src/types/events/base-events.ts

```diff
@@ -640,6 +644,68 @@ export interface UserChangeEvent {
   };
 }
 
+export interface WorkflowDeletedEvent {
+  type: 'workflow_deleted';
+  workflow_id: string;
+  workflow_draft_configuration: {
+    version_id: string;
+    app_steps: {
+      app_id: string;
+      workflow_step_id: string;
+      callback_id: string;
+    }[];
+  };
+  event_ts: string;
+}
+export interface WorkflowPublishedEvent {
```

Comment:
Nit: All other interfaces are separated by a newline

```suggestion
}

export interface WorkflowPublishedEvent {
```